checkstyle/suppressions.xml

```diff
@@ -39,6 +39,7 @@
     <suppress checks="(NPathComplexity|ClassFanOutComplexity|CyclomaticComplexity|ClassDataAbstractionCoupling|FinalLocalVariable|LocalVariableName|MemberName|ParameterName|MethodLength|JavaNCSS|AvoidStarImport)"
               files="core[\\/]src[\\/](generated|generated-test)[\\/].+.java$"/>
     <suppress checks="NPathComplexity" files="(ClusterTestExtensions|KafkaApisBuilder|SharePartition).java"/>
+    <suppress checks="CyclomaticComplexity" files="(SharePartition).java"/>
```

Comment:
Can we reduce the cyclomatic complexity? I have refactored acquire and acknowledge methods to sub methods to decrease the complexity.